docs/guides.md

```diff
@@ -0,0 +1,107 @@
+# Vaultwarden Setup Guide and Integrations
+
+Please note that the version of Bitwarden server used on the Embassy is a fork called "Vaultwarden."  Despite this, it still functions with several of the apps/integrations that are built for, and still named after, "Bitwarden."  We apologize for any confusion this may cause.
+
+## Vaultwarden Vault (WebUI)
+
+1. First we will open up our Embassy and go to the Marketplace tab, select Vaultwarden, and install if you have not already done so.
+
+2. After install, click Configure.  A strong admin token has been securely created by your Embassy, but you may change it if you wish.  This can also be changed later.  Click save.  Click start to run the service.
+
+3. Next launch the Vaultwarden web interface. If you are connected to your Embassy on LAN, Vaultwarden will also launch on LAN. If you are connected over Tor, Vaultwarden will lanuch on Tor. What you are looking at now is your very own Vaultwarden website, served right from your Embassy. Let's create an account. Please note, you are creating an account with yourself on your own Vaultwarden website served from your own Vaultwarden server, there are no third parties involved here.  Enter an email for logging in, a strong password that you will remember, and optionally a password hint.  It is important to realize that this is the 'Master password' for all your other passwords.  Make it very strong, memorize it, write it down, and back it up to a safe place. If you lose it, you may lose access to all your passwords and your entire digital life.
+
+Now you can log in to your new password manager!
+
+4. We are greeted with our newly setup Vaultwarden password vault.  At the top you can visit 'Tools' for reports, a password generator, and to import data from another program, such as lastpass, onepass, or keepass.  'Settings' gives you all your options, including the ability to set up 2 factor authorization.
+
+5. Back at the main page for your vault, let’s do a quick example login.  We can setup a new folder here and save new login.  For our example, we’re going to use our FileBrowser instance.  So click ‘Add item,’  then we’ll fill in some test information.  A name for the entry, a username, password, and the website we want to log in to.  Then click ‘Save.’
+
+## Bitwarden Browser Extension
+
+The Bitwarden browser extension will only work with Tor Borwser or Firefox browser, since those are the only two browsers capable of using Tor, or in other words, access .onion URLs. Brave Browser can also access .onion URLs, but only in special tabs, not in extension, so the Bitwarden extension will not work.
```

Comment:
```suggestion
The Bitwarden browser extension will only work with Tor browser or Firefox browser, since those are the only two browsers capable of using Tor, or in other words, access .onion URLs. Brave Browser can also access .onion URLs, but only in special tabs, not in extension, so the Bitwarden extension will not work on Brave.
```